extensions/amp-form/0.1/amp-form.js

```diff
@@ -414,64 +426,70 @@ export class AmpForm {
         tryFocus(autofocus);
       }
     });
+    if (this.ampFormService_) {
```

Comment:
Short-circuit instead. Makes the diff cleaner and the code more shallow.

```suggestion
    if (!this.ampFormService_) {
      return;
    }
```

But it should absolutely be available, so even better:

```suggestion
    devAssert(this.ampFormService_);
```